decidim-core/app/views/decidim/participatory_process_steps/_step.html.erb

```diff
@@ -0,0 +1,21 @@
+<% # TODO: Move to decorator? %>
+<% status_class = step.end_date > Time.zone.now  ? "timeline__item--inactive" : "" %>
+<% status_class = step.active? ? "timeline__item--current" : status_class %>
+
+<li class="<%= "timeline__item #{status_class}" %>">
+  <div class="timeline__phase">
+    <span class="timeline__phase__number"><%= step.position %></span>
+  </div>
+  <div class="timeline__info">
+    <div class="timeline__header">
+      <span class="timeline__date text-small">
+        <%= step.start_date.strftime("%d/%m/%Y") %> - <%= step.end_date.strftime("%d/%m/%Y") %>
```

Comment:
:+1: For localizing. Also, 

I'm using `end_date` as a condition to style old and upcoming steps, which may be wrong from scratch, which leads to: how should we style them if they can have null `end_date`s?